cli/command/engine/activate.go

```diff
@@ -112,7 +126,7 @@ func runActivate(cli command.Cli, options activateOptions) error {
 			client := cli.Client()
 			_, err := client.Ping(ctx)
 			return err
-		}); err != nil {
+		}, clitypes.RuntimeMetadataDir); err != nil {
```

Comment:
Related to my other comments about adding this option to the `clitypes.EngineInitOptions`; it looks like currently we only pass the default value.

We could get that default in `client.ActivateEngine()` /  `client.DoUpdate()`, i.e., something like

```go
func (c *baseClient) DoUpdate(ctx context.Context, opts clitypes.EngineInitOptions, out clitypes.OutStream,
	authConfig *types.AuthConfig, healthfn func(context.Context) error) error {
	if opts.MetaDataDir == "" {
		opts.MetaDataDir = clitypes.RuntimeMetadataDir
	}
	...
}
```

Doing so, makes the default (empty string) a useful value